projects/schematics/package.json

```diff
@@ -13,8 +13,8 @@
   "scripts": {
     "build": "npm run clean && npm run copy-feature-toggles && npm --prefix ../../ run generate:deps && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
     "clean": "../../node_modules/.bin/rimraf --glob \"index.js*\" \"index.d.ts\" \"src/**/*.js\" \"src/**/*.js.map\" \"src/**/*.d.ts\"",
-    "test": "npm run clean && npm run copy-feature-toggles && ../../node_modules/.bin/jest --config ./jest.schematics.config.js",
-    "copy-feature-toggles": "node copy-feature-toggles"
+    "copy-feature-toggles": "node copy-feature-toggles",
+    "test": "npm run clean && npm run copy-feature-toggles && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
```

Comment:
Is this change _really_ related to this PR?

Reply:
Thanks, good catch! I ran the `config:update` command that formats `package.json` files and must have accidentally omitted this.